selenium/automate_chrome.sh

```diff
@@ -3,28 +3,54 @@ set -e
 input=$1
 driver_version=$2
 tag=$3
+channel=${4:-"stable"}
 
 if [ -z "$1" -o -z "$2" -o -z "$3" ]; then
-    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version>'
+    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version> [{stable|dev|beta}]'
```

Comment:
Do we need another parameter [{stable|dev|beta}] for tagging?

Reply:
Changed